dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphUtility/BarGraph.cs

```diff
@@ -23,62 +23,59 @@
+        private Compositor _compositor;
+        private IntPtr _hwnd;
 
-        private float graphWidth, graphHeight;
-        private float shapeGraphContainerHeight, shapeGraphContainerWidth, shapeGraphOffsetY, shapeGraphOffsetX;
-        private float barWidth, barSpacing;
-        private float maxBarValue;
+        private float _graphWidth, _graphHeight;
+        private float _shapeGraphContainerHeight, _shapeGraphContainerWidth, _shapeGraphOffsetY, _shapeGraphOffsetX;
+        private float _barWidth, _barSpacing;
+        private double _maxBarValue;
 
-        private GraphBarStyle graphBarStyle;
-        private List<Windows.UI.Color> graphBarColors;
+        private GraphBarStyle _graphBarStyle;
+        private Windows.UI.Color[] _graphBarColors;
 
-        private Hashtable barValueMap;
+        private Bar[] _bars;
 
-        private WindowRenderTarget textRenderTarget;
-        private SolidColorBrush textSceneColorBrush;
-        private TextFormat textFormatTitle;
-        private TextFormat textFormatHorizontal;
-        private TextFormat textFormatVertical;
+        private WindowRenderTarget _textRenderTarget;
+        private SolidColorBrush _textSceneColorBrush;
+        private TextFormat _textFormatTitle;
+        private TextFormat _textFormatHorizontal;
+        private TextFormat _textFormatVertical;
 
-        private ShapeVisual shapeContainer;
-        private CompositionLineGeometry xAxisLine;
-        private CompositionLineGeometry yAxisLine;
-        private ContainerVisual mainContainer;
+        private ShapeVisual _shapeContainer;
+        private CompositionLineGeometry _xAxisLine;
+        private CompositionLineGeometry _yAxisLine;
+        private ContainerVisual _mainContainer;
 
-        private static SharpDX.Mathematics.Interop.RawColor4 black = new SharpDX.Mathematics.Interop.RawColor4(0, 0, 0, 255);
-        private static SharpDX.Mathematics.Interop.RawColor4 white = new SharpDX.Mathematics.Interop.RawColor4(255, 255, 255, 255);
+        private static SharpDX.Mathematics.Interop.RawColor4 _black = new SharpDX.Mathematics.Interop.RawColor4(0, 0, 0, 255);
+        private static SharpDX.Mathematics.Interop.RawColor4 _white = new SharpDX.Mathematics.Interop.RawColor4(255, 255, 255, 255);
 
-        private static float textSize = 20.0f;
+        private static float _textSize = 20.0f;
 
-        private AmbientLight ambientLight;
-        private SpotLight barOutlineLight;
-        private PointLight barLight;
+        private AmbientLight _ambientLight;
+        private SpotLight _barOutlineLight;
+        private PointLight _barLight;
 
-        #region public setters
         public string Title { get; set; }
         public string XAxisLabel { get; set; }
         public string YAxisLabel { get; set; }
         public ContainerVisual BarRoot { get; }
         public ContainerVisual GraphRoot { get; }
-        #endregion
 
         public enum GraphBarStyle
         {
```

Comment:
Don't give enums values unnecessarily. I don't think you care about what each value is.

Reply:
updated